client/gutenberg/editor/style.scss

```diff
@@ -7,10 +7,10 @@
 	left: 0;
 	background-color: $white;
 	pointer-events: none;
-	z-index: z-index( 'root', '.is-group-editor::before' );
+	z-index: z-index( 'root', '.is-group-gutenberg-group::before' );
```

Comment:
Do we need this?

`.is-group-editor::before` is defined in the _z-index list:
https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/shared/functions/_z-index.scss#L36

But `.is-group-gutenberg-group::before` isn't, so I'm pretty sure this doesn't do anything. 🤔 
https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/shared/functions/_z-index.scss#L294

Reply:
I just did a replace on existing usages - maybe we should update the one in `_z-index` list too?

Reply:
Added the new value in `_z-index` list in 8b6f741883d710c5c01b80bb6a61d11f2ff5b494.